sound/soc/codecs/rt711-sdw.c

```diff
@@ -443,6 +443,13 @@ static int rt711_sdw_probe(struct sdw_slave *slave,
 {
 	struct regmap *sdw_regmap, *regmap;
 
+	/* filter BIOS work-around on some Intel platforms */
```

Comment:
I had to read the above comment a few times. This isn't "filtering a workaround", right? Maybe "/* catch faulty BIOS configuration on some Intel platforms */"

Reply:
it's really 'only allow for BIOS workaround in specific configurations'

Reply:
fixed now